### ISSUE TITLE
Export Sketch or Proposal to Prototyping Tool

### ISSUE THREAD

**Issue by parabolika66** [MyrionPhoenixmoon/scenarioo#36]
As a user I want to export sketches from Scenarioo to a more powerful tool so that I can build high fidelity mock-ups.

Comments:
- Photoshop was mentioned as a target tool.
- Balsamiq offers a standard import / export format.

**Comment by adiherzog** [comment]
I don't know whether exporting a sketch / proposal to edit it in a different tool is that important. Such an export would only be useful if the sketched objects would be placed in different layers in Photoshop and would still be editable objects in Balsamiq. If it's just a flat image, it's probably of no use.

Was this workflow (edit in simple tool first and then edit in more sophisticated tool) ever mentioned in the interviews? I would assume that somebody would usually choose either the simple or the sophisticated tool and then stick to it for one issue.

It's probably more important that an existing Scenario can be exported so that it can be used as the base of a new Balsamiq / Axure / etc. mockup.

**Comment by MyrionPhoenixmoon** [comment]
Yes, this workflow was mentioned, which is why we wanted this to be recorded.
We agree though, that the Export of a complete Scenario or Proposal as a .zip is the highest priority export function. Even that is still not very high prio overall.
